### DISCUSSION TITLE
开启proxy后，视频文件直链VLC无法播放

### DISCUSSION THREAD

**Discussion by zzwk** [AlistGo/alist#1026] [Q&A]
使用的是cloudflare worker，开启proxy后访问链接正常302跳转到worker域名开始下载，其他播放器可以播放，VLC提示无法播放。关闭proxy则VLC可以直接播放。
似乎也并非是302跳转导致的问题，直接拿302跳转的目标链接测试，也同样是可以下载但无法VLC播放。我猜测大概率是worker改动了什么信息导致视频无法播放了。
VLC提示和日志如图：
![image](https://user-images.githubusercontent.com/51788980/166144797-1f4b2b6a-432b-486a-b766-b6dea607e7da.png)
![image](https://user-images.githubusercontent.com/51788980/166144825-e116a256-7d63-427e-92bb-a5c4219fb9cc.png)

使用302跳转的目标链接测试：
![image](https://user-images.githubusercontent.com/51788980/166145789-f73611fd-1d2b-4174-9058-f1532f302f52.png)
![image](https://user-images.githubusercontent.com/51788980/166145804-12fb939d-83fb-4501-8364-4121808fdad8.png)
测试文件链接：https://alist.mowan.ml/d/google/%5BNekomoe%20kissaten%5D%5BSummer%20Time%20Rendering%5D%5B01%5D%5B1080p%5D%5BJPSC%5D.mp4

我明白这大概率是VLC的锅，但是才疏学浅实在没法自己在worker的js脚本里找到怎么解决，实在是麻烦各位大大了…

**Comment by xhofe** [comment]
down_proxy_url 去掉后面的 `/`

**Reply by zzwk** [reply]
感谢回复！刚刚尝试了去掉/，依然提示无法播放，302以后的链接也无法播放。下载依然正常。目前测试链接的跳转已经去掉/了。

**Reply by xhofe** [reply]
那我不太清楚了 我使用IINA打开正常
<img width="996" alt="image" src="https://user-images.githubusercontent.com/36558727/166146785-ab241bf4-0f80-4029-a83c-27756bb90c3d.png">

**Reply by zzwk** [reply]
确实，nplayer、MXplayer也正常，就只有VLC无法打开，原因不明。不过还是感谢作者大大！
